components/Snackbar.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <v-snackbar v-model="model" :color="color" bottom left>
```

Comment:
You can probably add the `app` prop here, so that it respects the layout of the various app components (Navigation drawer, footer, app bar, etc).

It's especially important once we set up a mini player in the app footer.